core/Inst.hpp

```diff
@@ -32,6 +32,32 @@ namespace olympia
     class Inst {
     public:
 
+        class RenameData{
+            public:
+            void setDestination(unsigned int destination){
+                dest = destination;
+            }
+            void setOriginalDestination(unsigned int destination){
+                original_dest = destination;
+            }
+            void setSource(std::pair<unsigned int, bool> source){
+                src.push_back(source);
+            }
+            std::vector<std::pair<unsigned int, bool>> getSource(){
+                return src;
+            }
+            unsigned int getDestination(){
+                return dest;
+            }
+            unsigned int getOriginalDestination(){
+                return original_dest;
+            }
```

Comment:
These methods need to be `const`:
```suggestion
            const std::vector<std::pair<unsigned int, bool>> & getSource() const {
                return src;
            }
            uint32_t getDestination() const {
                return dest;
            }
            uint32_t getOriginalDestination() const {
                return original_dest;
            }
```